substrate/utils/frame/benchmarking-cli/src/storage/mod.rs

```diff
@@ -21,3 +21,22 @@ pub mod template;
 pub mod write;
 
 pub use cmd::StorageCmd;
+
+/// Empirically, the maximum batch size for block validation should be no more than 10,000.
+/// Bigger sizes may cause problems with runtime memory allocation.
+pub(crate) const MAX_BATCH_SIZE_FOR_BLOCK_VALIDATION: usize = 10_000;
+
+pub(crate) fn get_wasm_module() -> Box<dyn sc_executor_common::wasm_runtime::WasmModule> {
+	use frame_storage_access_test_runtime::WASM_BINARY;
+	use polkadot_node_core_pvf_common::executor_interface::DEFAULT_CONFIG;
```

Comment:
Unfortunately, this default configuration uses only `const EXTRA_HEAP_PAGES: u32 = 2048`  which is different from what the relay-chain has ` MaxMemoryPages: 8,192`, so on polkadot-asset-hub state with a batch of 10_000 this crashes with:

```
Thread 'main' panicked at 'Failed to allocate memory: "Allocator ran out of space"', substrate/primitives/io/src/lib.rs:1567
```

I think we should set maximum_pages to 8192 here.